### DISCUSSION TITLE
Is there a better On Screen Keyboard?

### DISCUSSION THREAD

**Discussion by lsitongia** [linux-surface/linux-surface#815] [Q&A] [answered]
It would be nice to have an On Screen Keyboard that has the control and alt keys, cursor keys, at least.

**Comment by qzed** [comment, marked as answer]
Assuming you're using gnome, there's an extension for alt/ctrl keys: https://extensions.gnome.org/extension/4413/improved-osk/.

**Reply by lsitongia** [reply]
Thank you. That right shift key is odd. Big improvement.
